### ISSUE TITLE
delete agents problem

### ISSUE THREAD

**Issue by AleksanderKamienski** [Tugcga/Path-Finder#13]
Hi, I there is a problem in your code in pathfinder/pyrvo/rvo_simulator.py line 38.

You schould change your code like this:

        for i in range(len(agent_indexes)):
            agent_index = agent_indexes[len(agent_indexes) - 1 - i]
            self._agents.pop(agent_index)
        self._force_update_agents = True
        
        into:
        
        sorted_agent_indexes = sorted(agent_indexes, reverse=True):
        for i in range(len(sorted_agent_indexes)):
            agent_index = sorted_agent_indexes[len(agent_indexes) - 1 - i]
            self._agents.pop(agent_index)
        self._force_update_agents = True
        

otherwise, when the indexes are sorted in ascending order, an error may occur when, for example, the list self._agents has 100 elements and the list agent_indexes will be [5,99]. This will make the call to self._agents.pop(99) fail because we will remove the agent with ID 5 before and the maximum index will be 98.

**Comment by Tugcga** [comment]
You are right, but it's not a bug, it's a feature. This method assumed that the input ```agent_indexes``` list is ordered in increasing order. It's mentioned in the description of the wrapper method in ```pathfinder/pyrvo/__init__.py:64```. Of course, it's possible to include the sort step into ```delete_agents``` method, but in some cases the input list may be already sorted, and then this step is unnecessary. I think that it's better to control the list of indexes by the user, and apply the sort step himself when it needs.

**Comment by AleksanderKamienski** [comment]
You are right. :)
